File: 2018/day_1/day_one.cpp
#include <iostream>
#include <fstream>
#include <string>
#include <cmath>

std::ifstream file;

int main(){
    file.open("input.txt", std::ifstream::in);
    
    std::string line;
    int result = 0;

    while(std::getline(file, line)){
        if(line[0] == '+')
            result += std::stoi(line.substr(1, line.length()-1));
        else if(line[0] == '-')
            result -= std::stoi(line.substr(1, line.length()-1));
    }
    std::cout << "Result: " << result << std::endl;

    return 0;
}

File: 2018/day_1/day_one.py
result = 0

file = open("/home/alexander/programmering/advent_of_code/2018/day_1/input.txt", "r")

for row in file:
    operator = row[0:1]
    if operator == '+':
        value = row.split('+')
        result += int(value[1])
    elif operator == '-':
        value = row.split('-')
        result -= int(value[1])

print("The result is: " + str(result))